src/Core/src/Handlers/RefreshView/RefreshViewHandler.Windows.cs

```diff
@@ -60,6 +60,34 @@ public static void MapRefreshColor(IRefreshViewHandler handler, IRefreshView ref
 		public static void MapRefreshViewBackground(IRefreshViewHandler handler, IView view)
 			=> UpdateBackground(handler);
 
+		public static void MapIsRefreshEnabled(IRefreshViewHandler handler, IRefreshView refreshView)
+			=> (handler as RefreshViewHandler)?.UpdateIsRefreshEnabled();
+
+		public static void MapIsEnabled(IRefreshViewHandler handler, IRefreshView refreshView)
```

Comment:
We don't need this method at all since the base already has it and it is the default.